be/database/migrations/2022_11_11_021215_create_activity_logs_table.php

```diff
@@ -15,7 +15,7 @@ public function up()
     {
         Schema::create('activity_logs', function (Blueprint $table) {
             $table->id();
-            $table->string('loggable_id');
+            $table->string('loggale_id');
```

Comment:
- do not change existing migrations (from the dev branch). Instead generate a new migration.
- `loggable_id` should be `unsignedBigInteger`